### PR TITLE
Wfly 15459 jakarta namespace saaj impl

### PR DESCRIPTION
**Pull request by rsearls**
https://issues.redhat.com/browse/WFLY-15459

### CODE REVIEW

File: ee-9/feature-pack/pom.xml [hunk 1]
```diff
@@ -939,6 +939,11 @@
             <type>pom</type>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.sun.xml.messaging.saaj</groupId>
+            <artifactId>saaj-impl</artifactId>
+        </dependency>
```
Comment (bstansberry) [inline ee-9/feature-pack/pom.xml:946]: Please move this one to ee-9/common/pom.xml, after the com.sun.istack one at https://github.com/wildfly/wildfly/blob/main/ee-9/common/pom.xml#L114.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Thanks, @rsearls. Can you rebase this on main to fix the merge conflict and eliminate the first 3 commits, which should be in main now?

File: ee-9/feature-pack/pom.xml (hunk 1 above)
Comment (rsearls) [reply ee-9/feature-pack/pom.xml]: I did rebase the branch before submitting the initial changes.  Resolving the conflicts required accepting the version changes in ee-9/pom.xml.

I moved saaj from feature-pack to common.

**Comment by rsearls** [comment]
prepare a new cleaner PR for this  issue

**Pull request closed without merging**